### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by uno36**
### Project requirements
-  **Important:** do NOT use `rails generate scaffold` command. The point of this project is to show you how to set up controllers by yourself.
-  For each URL you should create:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-  Make sure that you added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    @user = User.find(params[:id])
+    @user.destroy
+    redirect_to users_path
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :bio, :photo)
+  end
```
Comment (julie-ify) [inline app/controllers/users_controller.rb]: - [ ] These controller actions have no routes. Kindly remove them. Kindly stick to the requirements. We only need actions for the post and user index and show routes.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    @post = Post.find(params[:id])
+    @post.destroy
+    redirect_to posts_path
+  end
+
+  private
+
+  def post_params
+    paramst.require(:post).permit(:author_id)
+  end
```
Comment (julie-ify) [inline app/controllers/posts_controller.rb]: - [ ] Kindly remove these controller actions.

File: app/controllers/comments_cotroller.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class CommentsController < ActionController::Base
```
Comment (julie-ify) [inline app/controllers/comments_cotroller.rb]: - [ ] Kindly remove this controller. We only need posts and users controllers. Plus there is no route for this controller, which we also don't need.

File: app/controllers/likes_controller.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class LikesController < ActionController::Base
```
Comment (julie-ify) [inline app/controllers/likes_controller.rb]: - [ ] Kindly remove this controller. We only need posts and users controllers. Plus there is no route for this controller, which we also don't need.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @uno36,

Good job so far!

## Highlights!
- Good code organization and routes ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @uno36 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All requested changes implemented

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**